core/test/src/mill/main/ParseArgsTest.scala

```diff
@@ -21,10 +22,11 @@ object ParseArgsTest extends TestSuite {
         )
       }
 
-      'empty - check(input = Seq.empty,
-                     expectedSelectors = Seq.empty,
-                     expectedArgs = Seq.empty,
-                     expectedIsMulti = false)
+      'empty - check(
+        input = Seq.empty,
+        expectedSelectors = Seq.empty,
+        expectedArgs = Seq.empty,
+        expectedIsMulti = false)
```

Comment:
Let's make scalafmt put closing parens on a new line. I believe there is a config option for this